test/python/test_host_factory_api.py

```diff
@@ -0,0 +1,93 @@
+from __future__ import absolute_import
+
+import unittest
+import datetime
+
+import openapi_client
+
+from . import api_config
+
+TEST_HOST = "testHost"
+HOST_FACTORY = "dev:host_factory:testFactory"
+FACTORY_POLICY = '''
```

Comment:
Elsewhere in your tests, you declare multiline strings like this:
```suggestion
FACTORY_POLICY = f'''
```

Reply:
This doesn't actually have to do with multi-line strings. It is a Python `f` [string](https://www.python.org/dev/peps/pep-0498/)